docs/src/jll.md

```diff
@@ -282,3 +282,24 @@ Some comments about how to write this file:
   you want to use to override the products of the JLL package must have the same
   tree structure as the artifact.  In our example we need to use `/usr` to
   override Fontconfig and `/usr/local` for Bzip2.
+
+### Overriding specific products
+
+Instead of overriding the entire artifact, you can override a particular library
```

Comment:
I'd say something like
```suggestion
Instead of overriding the entire artifact, you can override a particular product (library, executable, etc...)
```